snapshotter/config/config.go

```diff
@@ -43,7 +43,11 @@ type listener struct {
 }
 
 type dialer struct {
-	AckMsgTimeoutInSeconds int `toml:"ack_msg_timeout_in_seconds" default:"1"`
+	DialTimeout       string `toml:"dial_timeout" default:"100ms"`
+	RetryTimeout      string `toml:"retry_timeout" default:"10s"`
+	RetryInterval     string `toml:"retry_interval" default:"5s"`
```

Comment:
This might be too slow but we have the knobs now and can tune based on observations.

Reply:
Regarding the timeouts, if we don't have them, would it wait indefinitely without clear errors? I wonder whether we could have a single timeout, instead of asking end-users to budget all of them specifically.

Reply:
I have not done any experiments here to prove without the timeouts. I agree reducing complexity would be better as long as the solution becomes stable.